app/controllers/admin/druids_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Admin
+  # Search and display a druid for a collection or work
+  class DruidsController < ApplicationController
+    include Dry::Monads[:result]
+
+    before_action :authenticate_user!
+    verify_authorized
+
+    def index
+      authorize! :admin_dashboard
+      return unless params[:query]
+
+      result = lookup_druid(params[:query])
+      return @failure = true if result.failure?
+
+      item = result.value!
+      redirect_to collection_version_path(item.head) if item.instance_of?(Collection)
```

Comment:
Can't use polymorphic_path since we need to redirect to the collection_version_path instead of collection_path (which would go to the settings tab). Is there some better way to do this under the circumstances?

Reply:
I would have done:
```ruby
redirect_to item.is_a?(Collection) ? item.head : item
```